addons/l10n_ch/static/src/scss/report_swissqr.scss

```diff
@@ -1,21 +1,21 @@
-body {
-    padding: 0!important;
+body.l10n_ch_qr {
+    padding: 0;
 
     /* Disable custom bakground */
     .o_report_layout_background {
         background: none;
         min-height: 0;
     }
+}
 
-    .swissqr_title {
-        position: absolute;
-        padding: 15px;
-        padding-top: 200px;
-    }
+.swissqr_title {
+    position: absolute;
+    padding: 15px;
+    padding-top: 150px;
```

Comment:
Why did this change?

Reply:
@william-andre It was originaly 150px. I changed it to 200px to prevent if we have a long street name + street2 + vat + ....
We had a similar issue with the header of invoices, and we conclude that the customer must increase the size in the paperformat. 

The swiss QR-report is linked (by default) to `European A4 without borders` paperformat, and by increasing the header-spacing and/or margin-top, I wasn't able to display it nicely and not really user friendly.

Therefore, not sure if better to set it to 200px to prevent case of street2 + vat + ... ( it will have it limitation of course) or let it by default to 150px (who I think fits most of customers)... what do you think?

Reply:
> The swiss QR-report is linked (by default) to `European A4 without borders` paperformat, and by increasing the header-spacing and/or margin-top, I wasn't able to display it nicely and not really user friendly.

AFAIK, that is the right way to fix it though, or else not using `position: absolute`. It is a limitation of using `wkhtmltopdf`: there is no way of knowing the height of the header from inside the document.